packages/bits/src/lib/popover/popover-modal.component.ts

```diff
@@ -79,6 +80,8 @@ export class PopoverModalComponent implements AfterViewInit, OnInit, OnDestroy {
 
     private popoverModalSubscriptions: Subscription[] = [];
 
+    get role(): string { return this.backdrop ? "alertdialog" : "dialog"; }
```

Comment:
I think, that it can be applicable for popover-modal.
![image](https://user-images.githubusercontent.com/75837119/113847323-09359480-97a0-11eb-9cd1-f69c9aee0862.png)

https://www.w3.org/TR/wai-aria-1.1/#alertdialog

Also we can see the example here: https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/alertdialog.html

https://dequeuniversity.com/library/aria/simple-alert-dialog